workloads/network-perf-v2/run.sh

```diff
@@ -15,9 +15,20 @@ log "###############################################"
 log "Workload: ${WORKLOAD}"
 log "###############################################"
 
-timeout $TEST_TIMEOUT ./k8s-netperf --debug --metrics --all --config ${WORKLOAD} --search $ES_SERVER --tcp-tolerance ${TOLERANCE} --clean=true
-run=$?
-
+echo $WORKLOAD
+        if [ $WORKLOAD == iperf ];then
```

Comment:
@SachinNinganure There are extra space at the beginning of this line and the following lines.

Reply:
If no other configs to be added differently for each workload in the future, I think you can consider to combine the first 3 if conditions together into a single if condition.
```
timeout $TEST_TIMEOUT ./k8s-netperf --debug --metrics --all --$WORKLOAD --search $ES_SERVER --tcp-tolerance ${TOLERANCE} --clean=true
```

Reply:
Noted!  would incorporate this , will wait for all the review comments.